### ISSUE TITLE
ML-Benchmark prediction submission: [XGBoost]

### ISSUE THREAD

**Issue by rtomeidl** [WCRP-CORDEX/ml-benchmark#18]
### Institution ID

IDL

### Institution Name

Instituto Dom Luiz, Faculdade de Ciências da Universidade de Lisboa

### Emulator identifier (`emulator_id`)

XGBoost_IDL_v1

### Emulator Description

XGboost provides a distributed, open-source machine learning library that uses gradient boosted decision trees. 
Input data is standardized at gridbox level using mean/std in the training period.
For both precipitation and maximum temperature rmse is used as a loss function.
Most relevant model parameters, max_depth = 6, gamma = 0, eta = 0.05 and early stopping at 20 rounds.

### Hardware and Training Details

One node with one NVIDIA A100 GPU (40GB)
Training time from 6 to 48 hours depending on variable/emulation method (maximum of 1000 rounds).
Inference time roughly 2 hour per set.

### Stochastic/Probabilistic Output

no

### Reference URL

10.1016/j.uclim.2024.101982

### Repository for Reproducibility

https://github.com/rtomeidl/CordexML_XGBoost

**Comment by jgonzalezab** [comment]
Please fill out [this form](https://forms.gle/1goF329LCrWwdJdh9) so we can give you access to our cluster to transfer your submission.

**Comment by jgonzalezab** [comment]
HI @rtomeidl. I went through your submission and it looks like the dates are off for most of the data files (e.g. files that should cover 2041–2060 are covering 1961–1980 instead) Could you check and resubmit? When computing predictions, make sure to stick to the period of each corresponding set of predictors. Upload the fixed version as `XGBoost_IDL_v1_v2.zip`

Thanks!

**Comment by IDLGroupRG1** [comment]
Hey! 
Sorry the files are fine but i did mess up the units on the time dimension. I already correct the files and uploaded for both submissions.

**Comment by jgonzalezab** [comment]
Thanks for the fix @rtomeidl. Unfortunately there is still a file with the wrong dates (off by just a couple of years). Would you mind fixing it? I only need you to send this files (maybe in a zip file named something like <emulator_name>_fixed_file.zip). Thanks!!

```
XGBoost_IDL_v2/SA_Domain/Emulator_hist_future/mid_century/imperfect/Predictions_pr_tasmax_NorESM2-MM_2041-2060.nc
```

**Comment by IDLGroupRG1** [comment]
Hey! Uploaded v3... sorry found a error in the inference. Hopefully the dates are fine now!
